Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Application.cs

```diff
@@ -53,5 +53,30 @@ public static bool GetHandleControlUpdatesOnMainThread(this IPlatformElementConf
 			return config;
 		}
 		#endregion
+
+		#region DisableAccessibilityScalingForNamedFontSizes
+		public static readonly BindableProperty DisableAccessibilityScalingForNamedFontSizesProperty = BindableProperty.Create("DisableAccessibilityScalingForNamedFontSizes", typeof(bool), typeof(Application), false);
```

Comment:
I prefer positive conditions, even if the default value is true. double negation is confusing